brake/backends/cachebe.py

```diff
@@ -71,7 +78,11 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            current_count = counters[counter]
+            if isinstance(current_count, tuple):
```

Comment:
Yeah, it's a whole upgrade cycle, but you can skip it if you want to. It provides people who are more sensitive to faults in their systems a chance to avoid that. I'll make a branch that pulls in that logic and try to cut a release before merging this. 

I'll start on that so we can still get this out there today.